### PR TITLE
support rolling log by time

### PR DESCRIPTION
**Pull request by taoyuanyuan**
### Issues associated with this PR
#457 

### Sign the CLA
Make sure you have signed the [CLA](https://www.clahub.com/agreements/alipay/sofa-mosn)

### Solutions
1. roller可配置
2. 新增通过日期做日志轮滚
3. 增加测试用例
4. readme文档

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases.
And you need to show the ut result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result

### CODE REVIEW

File: pkg/log/logger.go [hunk 1]
```diff
@@ -318,6 +337,21 @@ func (l *logger) Fatalln(args ...interface{}) {
 	os.Exit(1)
 }
 
+// default roller by daily
+func (l *logger) Write(p []byte) (n int, err error) {
+	if !l.create.IsZero() {
```
Comment (ldxdl) [inline pkg/log/logger.go]: 这里判断的条件不太好理解，可以再明确一下条件含义

**Comment by ldxdl** [comment]
LGTM
